templates/discourse/files/deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+unset NPM_CONFIG_PREFIX
```

Comment:
You're running this in the build and the deploy?

Reply:
Yes, both build and runtime need to be shifted over into modern-node mode.  That's what our docs say to do.